src/Engine/Renderer/Mesh/Mesh.cpp

```diff
@@ -243,12 +243,14 @@ void Mesh::updateGL() {
         {
             if ( m_renderMode == RM_POINTS )
             {
+                m_numElements = m_mesh.vertices().size();

```

Comment:
@dlyr @MathiasPaulin do we want to do this update each time we render the mesh ?

An alternative would be to have a mechanism to update the GL object from it's `Core` version. I remember it is something we discussed, but I can figure out the current status.

Any remark/comment would help here.

Reply:
this update is not done each time we render the mesh, it is done only if we did set the `INDEX` as dirty

Reply:
Ok, my bad.